h/static/scripts/directive/timestamp.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var dateUtil = require('../date-util');
+
+// @ngInject
+function TimestampController($scope, time) {
+  var vm = this;
+
+  /** A fuzzy, relative (eg. '6 days ago') format of the annotation's
```

Comment:
Perhaps update the comments to not reference annotations specifically, now that this is a general purpose directive?